server/pyproject.toml

```diff
@@ -25,21 +25,18 @@ quart = {version = ">=0.17,<0.19", optional = true}
 Quart-CORS = {version = "^0.5.0", optional = true}
 hypercorn = {version = ">=0.13,<0.15", optional = true}
 pymongo = {version = ">=4.2.0", optional = true, extras = ["srv", "tls"]}
-snowflake-connector-python = {version = "^2.7.8", extras = ["pandas"], optional = true}
 psycopg = {optional = true, version = "^3.0.15"}
-google-cloud-bigquery = {version = "^3.3.0", optional = true, extras = ["pandas", "bqstorage"]}
-toucan-connectors = {version = "^3.17.4", optional = true, extras = ["awsathena"]}
+toucan-connectors = {version = "^4.0.0", optional = true, extras = ["awsathena", "google_big_query", "mongo", "Redshift", "snowflake"]}
 pytest-xdist = ">=2.5,<4.0"
 
 [tool.poetry.dev-dependencies]
 pytest-cov = "^4.0.0"
 pytest-mock = "^3.10.0"
 # required so we can use nosql_apply_parameters_to_query during tests
-toucan-connectors = { version = "^3.17.4", extras = ["awsathena"] }
+toucan-connectors = { version = "^4.0.0", extras = ["awsathena", "google_big_query", "mongo", "Redshift", "snowflake"] }
 pytest-benchmark = "^4.0.0"
 snowflake-sqlalchemy = "^1.4.4"
```

Comment:
We probably want to get rid of this one too ?

Reply:
Nope, it's used in the tests